### PR TITLE
Update cms-csv-files.xml

### LINKED ISSUE #795: CMS: add example CSV output file for SUSYBSMAnalysis-RazorFilter
As discussed in #764, it will be good to create a new open data record describing the output CSV file coming from SUSYBSMAnalysis-RazorFilter analysis.

@jmduarte [described](https://github.com/cernopendata/opendata.cern.ch/issues/764#issuecomment-121267208) the meaning of columns as:

The columns are described by the header:
Run,Lumi,Event,MR,Rsq,E1,Px1,Py1,Pz1,E2,Px2,Py2,Pz2,HT,MET,nJets,nBJets

Run,Lumi,Event are the run number, lumi section, and event number for this event.
MR and Rsq are the values of the razor variables.
E1,Px1,Py1,Pz1 is the four-vector of the leading megajet (with the largest transverse momentum).
E2,Px2,Py2,Pz2 is the four-vector of the subleading megajet (with the largest transverse momentum).
HT is the scalar sum of the transverse momentum of the jets.
MET is the magnitude of the vector sum of the transverse energy of the particles in the event.
nJets it the number of jets with transverse momentum above 40 GeV.
nBJets is the number of b-tagged jets with transverse momentum above 40 GeV.

@pherterich Can you please create a record for this output dataset? I'll then add FFT file as usual.

### PR DESCRIPTION
**Pull request by pherterich**
Should close #795
I took information on size and events from the file, most context is taken from http://opendata.cern.ch/record/700 when I thought it was appropriate. Let me know if anything should be added, deleted or rephrased.
If it's fine, @tiborsimko please FFT (and 8564?)

### CODE REVIEW

File: invenio_opendata/testsuite/data/cms/cms-csv-files.xml [hunk 1]
```diff
@@ -1,4 +1,96 @@
...
+    <subfield code="a">CERN-LHC</subfield>
+    <subfield code="e">CMS</subfield>
+  </datafield>
+  <datafield tag="581" ind1=" " ind2=" ">
+    <subfield code="a">Comma-Separated-Value (csv) files can be used in online applications such as the one found here</subfield>
+    <subfield code="u">http://opendata.cern.ch/visualise/histograms/CMS</subfield>
+  </datafield>
+  <datafield tag="772" ind1=" " ind2=" ">
+    <subfield code="a">/MultiJet/Run2010B-Apr21ReReco-v1/AOD</subfield>
+    <subfield code="o">10.7483/OPENDATA.CMS.QAJC.TR8V</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-csv-files.xml]: `772 $o` isn't in our data model.  Maybe `$w` is enough?

**Comment by tiborsimko** [comment]
Thanks, cherry-picked as 6c9bbcf060bbf40ba53619d05257ee95c5b2b051and added FFT handling.

File: invenio_opendata/testsuite/data/cms/cms-csv-files.xml [hunk 2]
```diff
@@ -1,4 +1,96 @@
...
+  <controlfield tag="001">554</controlfield>
+  <datafield tag="024" ind1="7" ind2=" ">
+    <subfield code="2">DOI</subfield>
+    <subfield code="a">10.7483/OPENDATA.CMS.GACK.GEJA</subfield>
+  </datafield>
+  <datafield tag="100" ind1=" " ind2=" ">
+    <subfield code="a">Duarte, Javier</subfield>
+  </datafield>
+  <datafield tag="245" ind1=" " ind2=" ">
+    <subfield code="a">Example CSV output file for SUSYBSMAnalysis-RazorFilter</subfield>
```
Comment (tiborsimko) [inline invenio_opendata/testsuite/data/cms/cms-csv-files.xml]: @jmduarte @katilp Is this the wanted title for the SUSYBSMAnalysis-RazorFilter example output CSV file record?

Comment (jmduarte) [inline invenio_opendata/testsuite/data/cms/cms-csv-files.xml]: Sounds good to me.

**Pull request closed without merging**